org.chromium.Chromium.yaml

```diff
@@ -0,0 +1,229 @@
+    autodelete: true
+
+  com.widevine.Widevine:
+    version: '1'
+    directory: widevine
+    autodelete: true
+
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node12
+  - org.freedesktop.Sdk.Extension.openjdk11
+
+modules:
+  - shared-modules/libsecret/libsecret.json
+
+  - name: libcups
+    make-args: [libs]
+    no-make-install: true
+    post-install:
+      - make install-headers install-libs
+    cleanup:
+      - /include
+    sources:
+      - type: archive
+        url: https://github.com/apple/cups/releases/download/v2.3.3/cups-2.3.3-source.tar.gz
+        sha256: 261fd948bce8647b6d5cb2a1784f0c24cc52b5c4e827b71d726020bcc502f3ee
+
+  - name: gtk-cups-backend
+    buildsystem: meson
+    make-args: [modules/printbackends/libprintbackend-cups.so]
+    no-make-install: true
+    post-install:
+      - install -Dm 755 modules/printbackends/libprintbackend-cups.so -t /app/lib/gtkmodules/3.0.0/printbackends/
+    sources:
+      - type: git
+        url: https://gitlab.gnome.org/GNOME/gtk.git
+        tag: 3.24.23
+
+  - name: gtk-settings
+    buildsystem: simple
+    build-commands:
+      - 'install -Dm 644 gtk-settings.ini /app/etc/xdg/gtk-3.0/settings.ini'
+    sources:
+      - type: file
+        path: gtk-settings.ini
+
+  # Based on https://github.com/flathub/shared-modules/blob/master/python2.7/python-2.7.json
+  - name: python2
+    config-opts:
+      - --enable-shared
+      - --with-ensurepip=yes
+      - --with-system-expat
+      - --with-system-ffi
+      - --with-dbmliborder=gdbm
+      - --enable-unicode=ucs4
+    build-options:
+      no-debuginfo: true
+    post-install:
+      - chmod 644 $FLATPAK_DEST/lib/libpython2.7.so.1.0
+    cleanup:
+      - '*'
+    sources:
+      - type: archive
+        url: https://www.python.org/ftp/python/2.7.18/Python-2.7.18.tar.xz
+        sha256: b62c0e7937551d0cc02b8fd5cb0f544f9405bafc9a54d3808ed4594812edef43
+
+  - name: xcbproto
+    build-options:
+      env:
+        PYTHON: /app/bin/python2
+    cleanup:
+      - '*'
+    sources:
+      - type: archive
+        url: https://xorg.freedesktop.org/archive/individual/proto/xcb-proto-1.14.tar.xz
+        sha256: 186a3ceb26f9b4a015f5a44dcc814c93033a5fc39684f36f1ecc79834416a605
+
+  - name: krb5
+    subdir: src
+    config-opts:
+      - --disable-static
+      - --disable-rpath
+    sources:
+      - type: archive
+        url: https://kerberos.org/dist/krb5/1.18/krb5-1.18.2.tar.gz
+        sha256: c6e4c9ec1a98141c3f5d66ddf1a135549050c9fab4e9a4620ee9b22085873ae0
+
+  - name: krb5-config
+    buildsystem: simple
+    build-commands:
+      - install -Dm 644 krb5.conf -t /app/etc
+    sources:
+      - type: file
+        path: krb5.conf
+
+  - name: extensions
+    buildsystem: simple
+    build-commands:
+      - mkdir -p /app/native-messaging-hosts /app/widevine
+
+  - name: chromium
+    buildsystem: simple
+    build-options:
+      env:
+        CC: clang
+        CXX: clang++
+        AR: ar
+        NM: nm
```

Comment:
Shouldn't these be `llvm-ar` and `llvm-nm`? I'm not sure if chromium is using some special feature in llvm's ar and nm but why risk it?

Reply:
Hmm, not entirely sure, but it definitely worked with standard ar and nm for my local builds, so I hadn't even though of using llvm-ar and llvm-nm.

Reply:
Fwiw, on Endless we build with `llvm-ar` and `nm` (non-llvm version)